src/components/IconSVG/ChangeColor.tsx

```diff
@@ -0,0 +1,16 @@
+const ChangeColor = ({ color }: { color: string }) => {
```

Comment:
Please hold this and let's integrate https://fonts.google.com/icons, it will be little bit different from design but we will have single source of truth and we don't need to create this kind of components

Reply:
Done